client/src/modules/Sidebar/components/ItemList/components/DefItem/EditDefItem/normalizeData.js

```diff
@@ -0,0 +1,15 @@
+const normalize = data => {
+  return {
+    title: data.title,
+    address: data.address,
+    floor: data.storage_place.match(/\d/)[0],
+    informational_plates: data.informational_plates,
+    phone: data.phone[0],
+    additional_information: data.additional_information,
+    storage_place: data.storage_place,
+    accessibility: data.accessibility,
+    coordinates: data.location.coordinates
+  };
```

Comment:
You can use destructuring and replace only floor property like { ...data, floor: data.storage_place.match(/\d/)[0]} where it's needed.